docs/data/stashcache/run-stash-origin-container.md

```diff
@@ -49,6 +51,25 @@ XC_RESOURCENAME=YOUR_SITE_NAME
 ORIGIN_FQDN=<FQDN>
 ```
 
+### Ensure the xrootd service has a certificate
```

Comment:
```suggestion
### Providing a host certificate
```

Talking about XRootD is one layer too deep, IMO, especially in a container setup. "I'm setting up a my OSDF origin container, what the heck's an xrootd!?"